### ISSUE TITLE
Include of angular-route.js and ngRoute module are missing

### ISSUE THREAD

**Issue by oliver-brm** [osgi/osgi.enroute.blog#2]
Hi,

although there is a description in the Angular.js documentation which JavaScript file and which service  to include, I would put a paragraph into the PDF file (on page 70-35) that tells readers to include angular-route.js and the ngRoute module. That might save some pain at that spot...

Besides, enRoute works really well and I really enjoy the work that was put in there. I will use it for the next application at my company (I was looking for a framework that is working in an OSGi container, anyway). And I have a feeling that this time, I will create a much better application than before.

Thanks for all the efforts!

Kind regards
Oliver

**Comment by pkriens** [comment]
Good idea. However, this blog example has to be rewritten since it was the first attempt. See enroute.osgi.org for the more modern tutorials. In the modern way, you don't have to worry anymore what to include, this is managed via annotations. I will mark this example.
